### ISSUE TITLE
ubuntu 20.10 breaks UTCClock

### ISSUE THREAD

**Issue by drhaz** [injcristianrojas/UTCClock#11]
Since update to Ubuntu 20.10, UTCClock is broken. Stack trace attached below (from Chrome gnome shell extension). Gnome Version is 3.28.1 

The settings of extension utcclock@injcristianrojas.github.com had an error:
```
Error: Schema org.gnome.shell.extensions.utcclock could not be found for extension utcclock@injcristianrojas.github.com. Please check your installation.
```

Stack trace:
```
getSettings@/home/dharbeck/.local/share/gnome-shell/extensions/utcclock@injcristianrojas.github.com/convenience.js:88:15
_init@/home/dharbeck/.local/share/gnome-shell/extensions/utcclock@injcristianrojas.github.com/prefs_window.js:20:38
wrapper@resource:///org/gnome/gjs/modules/script/_legacy.js:82:27
_init@/home/dharbeck/.local/share/gnome-shell/extensions/utcclock@injcristianrojas.github.com/prefs.js:29:22
wrapper@resource:///org/gnome/gjs/modules/script/_legacy.js:82:27
buildPrefsWidget@/home/dharbeck/.local/share/gnome-shell/extensions/utcclock@injcristianrojas.github.com/prefs.js:36:17
_init@resource:///org/gnome/Shell/Extensions/js/extensionsService.js:207:40
OpenExtensionPrefsAsync/<@resource:///org/gnome/Shell/Extensions/js/extensionsService.js:122:28
asyncCallback@resource:///org/gnome/gjs/modules/core/overrides/Gio.js:132:22
run@resource:///org/gnome/Shell/Extensions/js/dbusService.js:177:20
main@resource:///org/gnome/Shell/Extensions/js/main.js:19:13
run@resource:///org/gnome/gjs/modules/script/package.js:222:19
start@resource:///org/gnome/gjs/modules/script/package.js:206:8
@/usr/share/gnome-shell/org.gnome.Shell.Extensions:1:17
```

**Comment by injcristianrojas** [comment]
Fixed. Sorry for the inconvenience.

**Comment by gene1wood** [comment]
No problem, thank you! Fixed in version `14` with 219b45129520e05afc5ab2b1c491dc97fe4c02a9
